lib/runner/run.js

```diff
@@ -187,6 +187,9 @@ Runner.readPaths = function(testSource, opts, cb) {
   if (opts.tag_filter) {
     errorMessage.push('; using tags:', opts.tag_filter);
   }
+  if (opts.tags_filter) {
+    errorMessage.push('; using commonly required tags:', opts.tags_filter.join(', '));
```

Comment:
we should find a better wording to differentiate tags to multiple-tags filter. 
`using combined tags` or smth like that.